src/js/fs.js

```diff
@@ -263,26 +255,25 @@ fs.readFileSync = function(path) {
   return Buffer.concat(buffers);
 };
 
+fs.writeFile = function(path, outdata, callback) {
```

Comment:
I'd prefer to keep the below things defined as they were previously.

a) property name: data - because it looks ok for me and the change doesn't meet document.
b) checking callback: checkArgFunction - throws errors because callback is mandatory.